cmd/migrate.go

```diff
@@ -147,7 +147,14 @@ func migrateReset() error {
 
 func initMigrations(vipr *viper.Viper, logger log.Logger) (*migrate.Migrate, error) {
 	pgCfg := flags.NewPostgresConfig(vipr)
-	dbURL := fmt.Sprintf("postgres://%s:%s@%s:%s/%s?", pgCfg.User, pgCfg.Password, pgCfg.Host, pgCfg.Port, pgCfg.Database)
+	var userPass string
+	if pgCfg.Password != "" {
```

Comment:
Password is not a mandatory value since you can configure PG to allow internal connections or other similar rules based on source